tests/api/api.spec.ts

```diff
@@ -948,6 +949,15 @@ describe("@dataform/api", () => {
         "example_view"
       ]);
     });
+
+    it("times out after timeout period during compilation", async () => {
+      try {
+        await compile({ projectDir: "df/examples/redshift_never_finishes_compiling" });
+        fail("Compilation timeout Error expected.");
```

Comment:
why you need this `fail` ?

Reply:
It's to handle the case where `compile` (incorrectly) doesn't throw a timeout Error.